Dockerfile

```diff
@@ -12,10 +12,17 @@ RUN apt-get update && \
  && sed 's/# PassThroughPattern:.*this would allow.*/PassThroughPattern: .* #/' -i /etc/apt-cacher-ng/acng.conf \
  && rm -rf /var/lib/apt/lists/*
 
+ENV TINI_VERSION v0.14.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini.asc /tini.asc
+RUN gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7 \
+ && gpg --verify /tini.asc
+RUN chmod +x /tini
+
 COPY entrypoint.sh /sbin/entrypoint.sh
 RUN chmod 755 /sbin/entrypoint.sh
 
 EXPOSE 3142/tcp
 VOLUME ["${APT_CACHER_NG_CACHE_DIR}"]
-ENTRYPOINT ["/sbin/entrypoint.sh"]
+ENTRYPOINT ["/tini", "--", "/sbin/entrypoint.sh"]
```

Comment:
can you invoke `tini` from the `entrypoint.sh` (https://github.com/sameersbn/docker-apt-cacher-ng/blob/master/entrypoint.sh#L37) script instead?